xcell/mappers/utils.py

```diff
@@ -60,3 +60,18 @@ def get_DIR_Nz(cat_spec, cat_photo, bands, zflag,
         dndz_jk.append(n)
     dndz_jk = np.array(dndz_jk)
     return zz, dndz, dndz_jk
+
+
+def _beam_gaussian(ell, fwhm_amin):
+    sigma_rad = np.radians(fwhm_amin / 2.355 / 60)
+    return np.exp(-0.5 * ell * (ell + 1) * sigma_rad**2)
+
+
+def get_beam(nside, beam_info):
+    if beam_info is None:  # No beam
+        beam = np.ones(3*nside)
+    else:
+        ell = np.arange(3*nside)
+        beam = _beam_gaussian(ell, beam_info)
+        beam /= beam[0]  # normalize it
+    return beam
```

Comment:
OK, since we are doing this, I suggest we make this a bit more general. See my suggestion below and let me know what you think

```suggestion
    if beam_info is None:  # No beam
        beam = np.ones(3*nside)
    elif beam_info['type'] == 'Gaussian':  # Gaussian beam
        ell = np.arange(3*nside)
        beam = _beam_gaussian(ell, beam_info['FWHM_arcmin'])
        beam /= beam[0]  # normalize it
    else:
        raise NotImplementedError("Unknown beam type.")
    return beam
```

Reply:
The rationale is that `beam_info` will either be `None`, or it will be a dictionary with some information about the beam. In the simplest case (the only one we have right now), it will be a Gaussian beam that is completely defined by its FWHM. However, there are other maps (e.g. the Planck frequency maps) that have tabulated beams in FITS files, so we could include those in the future.

Reply:
Implemented